packages/js/product-editor/src/components/variations-table/pricing-menu-item/pricing-menu-item.tsx

```diff
@@ -434,6 +436,12 @@ export function PricingMenuItem( {
 						>
 							{ __( 'Schedule sale', 'woocommerce' ) }
 						</MenuItem>
+						<VariationQuickUpdateMenuItem.Slot
+							group={ 'pricing' }
+							supportsMultipleSelection={
+								supportsMultipleSelection
+							}
+						/>
```

Comment:
We need to pass through the `selection` and `onChange` so that fills can access that in their `onClick` handler and actually do something!

We should also pass  the`onClose` callbacks, as that will be needed too for extensions to close the menu when they have done their action!